storage/cmd/gcsemulator/gcsemulator.go

```diff
@@ -43,3 +47,44 @@ func main() {
 	fmt.Printf("Cloud Storage emulator running on %s\n", server.Addr)
 	select {}
 }
+
+//export Start
+func Start() {
+	opts := gcsemu.Options{
+		Verbose: false,
+		Log: func(err error, fmt string, args ...interface{}) {
+			if err != nil {
+				fmt = "ERROR: " + fmt + ": %s"
+				args = append(args, err)
+			}
+			log.Printf(fmt, args...)
+		},
+	}
+	server, err := gcsemu.NewServer("localhost:9000", opts)
+	if err != nil {
+		log.Fatalf("failed to start server: %s", err)
+	}
+
+	defer server.Close()
+
+	fmt.Printf("Cloud Storage emulator running on %s\n", server.Addr)
+	started <- 0
+	select {
+	case <-quit:
+		fmt.Println("Cloud Storage emulator exiting")
+	}
+}
+
+//export WaitForStart
+func WaitForStart() {
+	select {
+	case <-started:
+		fmt.Println("Cloud Storage emulator is started")
+	}
+}
+
+//export Stop
+func Stop() {
+	quit <- 0
+}
+
```

Comment:
What's all this?

Reply:
it is need by the javaclient to start and stop the gcs emulator from the java code where 
`import "C"` along with the export comments produce c library from the go code then called by jni from java to start/stop the server in java test cases

Reply:
removed